libkbfs/disk_block_cache_helper_test.go

```diff
@@ -0,0 +1,53 @@
+package libkbfs
+
+import (
+	"testing"
+	"time"
+
+	"github.com/keybase/kbfs/kbfscodec"
+	"github.com/keybase/kbfs/tlf"
+	"github.com/stretchr/testify/require"
+)
+
+var testMetaDb = DiskBlockCacheMetadata{
+	TlfID: tlf.FakeID(1, tlf.Public),
+	// Encodes as ... d5 ad 00 00 be ef.
+	LRUTime:          legacyEncodedTime{time.Unix(0xdead, 0xbeef)},
+	BlockSize:        0xface,
+	FinishedPrefetch: true,
+}
+
+// This was generated by starting with []byte{} in
+// TestDiskBlockCacheMetadataEncodeHardcoded and copying the data in
+// the error message for its require.Equal.
```

Comment:
Don't we need to test a `testMetaDb` generated _before_ the change?

Reply:
Yeap, if you look at the commit history, you'll see that I reverted the go-codec change, then generated the hardcoded data.